backend/src/bundles/auth/auth.service.ts

```diff
@@ -3,6 +3,7 @@ import {
     type UserSignUpResponseDto,
 } from '~/bundles/users/types/types.js';
 import { type UserService } from '~/bundles/users/user.service.js';
+import { token } from '~/common/token/token.js';
```

Comment:
As for me better do this class like some service. Another which we have currently in the project. So just keep all files the same but move token folder to src/common/services which will has services.ts with all services initialization and folder token which contains all current files. And change name of this class like TokenService, because we use it like the separate module, that's why for me much better make it like service.